src/main/java/stream/flarebot/flarebot/util/MessageUtils.java

```diff
@@ -180,12 +180,12 @@ public void run() {
         }.delay(delay);
     }
 
-    public static RestAction<Message> getUsage(Command command, TextChannel channel) {
+    public static RestAction<Message> getUsage(Command command, TextChannel channel, User user) {
         String title = capitalize(command.getCommand()) + " Usage";
         String usage = HelpFormatter.formatCommandPrefix(channel, command.getUsage());
         String permission = command.getPermission() + "\n" +
                 "**Default permission: **" + command.isDefaultPermission();
-        return channel.sendMessage(new EmbedBuilder().setTitle(title, null).addField("Usage", usage, false)
+        return channel.sendMessage(getEmbed(user).setTitle(title, null).addField("Usage", usage, false)
```

Comment:
Besides, doesn't make any difference to the actual PR. I have put it in the PR description so it is technically part of my PR